src/bosh-dns/dns/config/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	Address            string       `json:"address"`
 	Port               int          `json:"port"`
 	BindTimeout        DurationJSON `json:"timeout,omitempty"`
+	RecursorRetryCount int          `json:"recursor_retry_count,omitempty"`
```

Comment:
I don't think you've added this property to the job spec yet

Reply:
Yes, I will merge the PR from @FlorianNachtigall to get that. (He implemented all the stuff at the spec)